docs/source/examples/syncing-code-artifacts.rst

```diff
@@ -140,6 +116,32 @@ file_mount may be slow because they are processed by ``rsync``.  Use
 :ref:`SkyPilot bucket mounting <sky-storage>` to efficiently handle
 large files.
 
+.. _exclude-uploading-files:
+
+Exclude uploading files
+--------------------------------------
+By default, SkyPilot uses your existing :code:`.gitignore` and :code:`.git/info/exclude` to exclude files from syncing.
+
+Alternatively, you can use :code:`.skyignore` if you want to separate SkyPilot's syncing behavior from Git's.
+If you use a :code:`.skyignore` file, SkyPilot will only exclude files based on that file without using the default Git files.
+You can place a :code:`.skyignore` file under either your workdir or file_mounts. 
+
+:code:`.skyignore` follows RSYNC filter rules, e.g.
+
+.. code-block::
+
+  # Files that match pattern under ONE directory
+  /file.txt
+  /dir
+  /*.jar
+  /dir/*.jar
+
+  # Files that match pattern under ALL directories
+  *.jar
+  file.txt
+
+Do NOT use ``.`` to indicate local directory (e.g. ``./file``).
```

Comment:
```suggestion
Do _not_ use ``.`` to indicate local directory (e.g., instead of ``./file``, write ``/file``).
```
Please check if this is what you meant?